lms/envs/common.py

```diff
@@ -1329,6 +1333,18 @@
         ],
         'output_filename': 'css/lms-style-xmodule-annotations.css',
     },
+    'style-edx-footer': {
+        'source_filenames': [
+            'sass/footer-v3.css',
+        ],
+        'output_filename': 'css/lms-style-edx-footer-v3.css',
+    },
+    'style-edx-footer-rtl': {
+        'source_filenames': [
+            'sass/footer-v3-rtl.css',
+        ],
+        'output_filename': 'css/lms-style-edx-footer-v3.css',
```

Comment:
Also, I think you need the ``*-rtl`` extension as part of this output file name, right?